bitcoin-core-dev-tech/2023-09/p2p-working-session.md

```diff
@@ -0,0 +1,204 @@
+---
+title: P2P working session
+categories: ['core-dev-tech']
+tags: ['bitcoin core', 'p2p']
+date: 2023-09-19
+---
+
+## Erlay
+
+- Gleb is active and ready to move forward - [#21515](https://github.com/bitcoin/bitcoin/pull/21515)
+- Are there people generally interested in review?
+  - I wanted first to convince myself that this is useful. I couldn't reproduce the numbers from the paper - 5% was what I got. 100 connections to erlay peers. My node is from a non-standard port. It may be that I don't have a normal sample. There is a pull request that could add RPC stats to bitcoind - that might get better numbers. We lose the stats once we disconnect.
```

Comment:
```suggestion
  - I wanted first to convince myself that this is useful. I couldn't [reproduce](https://github.com/vasild/bitcoin/wiki/Erlay) the numbers from the paper - 5% was what I got with ~100 connections. My node is listening on a non-standard port. It may be that I don't have a normal sample. There is a [pull request](https://github.com/bitcoin/bitcoin/pull/27534) that could add RPC stats to bitcoind - that might get better numbers. Current stats are per peer and we lose those stats once the peer disconnects.
```